hack/testing/rsyslog/roles/openshift_logging_rsyslog/tasks/setup_certs.yaml

```diff
@@ -0,0 +1,106 @@
+
+- name: install CA cert file
+  copy: content={{ hostvars[groups['masters'][0]]['fluentdcak8s'] }} dest={{rsyslog__viaq_path}}/mmk8s.ca.crt mode=0400
+
+- name: install ES CA cert file
+  copy: content={{ hostvars[groups['masters'][0]]['fluentdcaes'] }} dest={{rsyslog__viaq_path}}/es-ca.crt mode=0400
+
+- name: install ES client cert file
+  copy: content={{ hostvars[groups['masters'][0]]['fluentdcert'] }} dest={{rsyslog__viaq_path}}/es-cert.pem mode=0400
+
+- name: install ES client key file
+  copy: content={{ hostvars[groups['masters'][0]]['fluentdkey'] }} dest={{rsyslog__viaq_path}}/es-key.pem mode=0400
+
+- name: setup host alias for es ip
+  shell: |
+    if grep -q '^{{ hostvars[groups['masters'][0]]['esip'] }} .* logging-es$' ; then
+      echo already have alias logging-es for {{ hostvars[groups['masters'][0]]['esip'] }}
+    else
+      sudo sed -i '/^{{ hostvars[groups['masters'][0]]['esip'] }}/d' /etc/hosts
+      sudo sed -i '/ logging-es$/d' /etc/hosts
+      echo {{ hostvars[groups['masters'][0]]['esip'] }} logging-es | sudo tee -a /etc/hosts
+    fi
+  when: elasticsearch_server_host is not defined
+
+- name: setup host alias for es-ops ip
+  shell: |
+    if grep -q '^{{ hostvars[groups['masters'][0]]['esopsip'] }} .* logging-es-ops$' ; then
+      echo already have alias logging-es-ops for {{ hostvars[groups['masters'][0]]['esopsip'] }}
+    else
+      sudo sed -i '/^{{ hostvars[groups['masters'][0]]['esopsip'] }}/d' /etc/hosts
+      sudo sed -i '/ logging-es-ops$/d' /etc/hosts
+      echo {{ hostvars[groups['masters'][0]]['esopsip'] }} logging-es-ops | sudo tee -a /etc/hosts
+    fi
+  when:
+  - elasticsearch_ops_server_host is not defined
+  - openshift_logging_use_ops | default(False)
+
+# Set by System policy ???
+- name: copy selinux policy files
+  copy: src={{ item }} dest=/root/ mode=0400
+  with_items:
+  - rsyslog2k8s.te
+  - rsyslog2es.te
+  - varlibdockercont.te
+  - rsyslogaccessnssdb.te
+  - rsyslogvar.te
+
+- name: build policy
+  shell: |
+    cd /root
+    sysmods=$( semodule -l | awk '{print $1}' )
+    for mod in varlibdockercont rsyslog2k8s rsyslogaccessnssdb rsyslog2es rsyslogvar ; do
+      if echo "$sysmods" | grep -q $mod ; then
+        echo using existing selinux module $mod
+      elif [ -f ${mod}.te ] ; then
+        checkmodule -M -m -o ${mod}.mod ${mod}.te
+        semodule_package -o ${mod}.pp -m ${mod}.mod
+        semodule -i ${mod}.pp
+      fi
+    done
+# Set by System policy ???
+
+- name: disable rsyslog
+  systemd:
+    name: rsyslog.service
+    enabled: no
+    state: stopped
+  when: use_rsyslog_image | default(False)
+
+- name: install rsyslog container systemd service
+  template: src=rsyslog-container.service.j2 dest=/usr/lib/systemd/system/rsyslog-container.service mode=0400
+  when: use_rsyslog_image | default(False)
+
+- name: enable and run rsyslog-container.service
```

Comment:
I think you are missing disabling rsyslog-container.service when: not use_rsyslog_image | default(False)

Reply:
Thanks!  Adding it...